### PR TITLE
hddl: make hddl daemon running in container by docker compose

### PR DESCRIPTION
**Pull request by alekdu**
* ov_hddl_init: build myd_ion and myd_vsc drivers for hddl
* ov_hddl_run: start hddl daemon

Once docker compose setup, the hddl container will always keep up.
This is compatible with both docker swam and k8s.

Signed-off-by: Alek Du <alek.du@intel.com>

### CODE REVIEW

**Comment by alekdu** [comment]
@xwu2git Yes, it will survive system reboot (startup along with docker service by systemd)

File: VCAC-A/script/setup_hddl.sh [hunk 1]
```diff
@@ -1,110 +1,136 @@
...
+if [ "\$1" == "init" ]; then
+        modprobe i2c-dev i2c-i801 i2c-hid
+        cd /opt/intel/openvino/deployment_tools/inference_engine/external/hddl/drivers
+        bash setup.sh install
+	while true; do sleep 3600; done
+else
+        while [ ! -c /dev/ion ] ; do sleep 1; done
+        source /opt/intel/openvino/bin/setupvars.sh
+        cd /opt/intel/openvino/deployment_tools/inference_engine/external/hddl/bin
+        while true; do ./hddldaemon; sleep 1; done
```
Comment (mythi) [inline VCAC-A/script/setup_hddl.sh]: kubernetes knows how to restart the service if it fails. Is this necessary for docker?

File: VCAC-A/script/setup_hddl.sh [hunk 2]
```diff
@@ -1,110 +1,136 @@
...
+{
+	cat > "$1" <<EOF
+#!/bin/bash
+
+if [ "\$1" == "init" ]; then
+        modprobe i2c-dev i2c-i801 i2c-hid
+        cd /opt/intel/openvino/deployment_tools/inference_engine/external/hddl/drivers
+        bash setup.sh install
+	while true; do sleep 3600; done
+else
```
Comment (mythi) [inline VCAC-A/script/setup_hddl.sh]: is the else branch possible to run without privileges?

File: VCAC-A/script/setup_hddl.sh (hunk 1 above)
Comment (alekdu) [reply VCAC-A/script/setup_hddl.sh]: Here, OVC also need support docker swarm mode, so I use docker-compose to create the service (not by kubelet). But yes, the service should bring it up always. Don't know if any people like to manually run the hddl container.

File: VCAC-A/script/setup_hddl.sh (hunk 2 above)
Comment (alekdu) [reply VCAC-A/script/setup_hddl.sh]: Before we fix the USB pipe write and I2C ioexpander write permission issue, the "else" part still need privileged. That's next goal.

**Pull request merged**